src/core/server/http/router/route.ts

```diff
@@ -18,7 +18,31 @@
  */
 
 import { ObjectType, Schema } from '@kbn/config-schema';
-export type RouteMethod = 'GET' | 'POST' | 'PUT' | 'DELETE';
+/**
+ * The set of common HTTP methods supported by Kibana routing.
+ * @public
+ * */
+export type RouteMethod = 'get' | 'post' | 'put' | 'delete';
```

Comment:
A couple questions:

- Why should these now be lower case? It looks like it would lead to less diff changes in this patch.
- Why not add `'patch' | 'options'` to this list since they are also used in: `method: RouteMethod | 'patch' | 'options';`

Reply:
- current codebase relays on method names in lower case, because it's a default in hapi. changing to the upper case would lead to bugs during migration.
- the router doesn't support PATCH and OPTIONS methods as of now and typing should reflect this.